### ISSUE TITLE
[Feature Request]: I hope that if you enable the lyrics sync feature, the default YouTube Music lyrics feature will be maintained.

### ISSUE THREAD

**Issue by sH1222J** [th-ch/youtube-music#2414]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

There are not many sync lyrics in the LRCLIB lyrics database.

### Proposed Solution

I would like to use YouTube's default lyrics in case there are no lyrics.

### Alternatives Considered

I wish they would maintain both the sync lyrics function and the built-in YouTube lyrics function.

**Comment by ArjixWasTaken** [comment]
Your stated problem is the lack of lyrics, but did you know that I am working on a PR to add more lyric sources since lrclib is lacking?

Yes, this would be a nice fallback if no lyrics were found, but it would be better to have more lyric sources first.

**Comment by LetrixZ** [comment]
> Your stated problem is the lack of lyrics, but did you know that I am working on a PR to add more lyric sources since lrclib is lacking?
> 
> Yes, this would be a nice fallback if no lyrics were found, but it would be better to have more lyric sources first.

I agree that more sources would be nice but in the meantime, would it be possible to fallback to Genius or regular lyrics?

I haven't checked how the plugin works and if it allows to be replaced once loaded.

**Comment by sH1222J** [comment]
> Your stated problem is the lack of lyrics, but did you know that I am working on a PR to add more lyric sources since lrclib is lacking?
> 
> Yes, this would be a nice fallback if no lyrics were found, but it would be better to have more lyric sources first.

If there's anything I can do to add more lyrics sources, I'd be happy to help, but I don't think it has anything to do with getting rid of YouTube's native features.

**Comment by ArjixWasTaken** [comment]
Maybe this can be closed?
It's not resolved since it is too specific, but it's basically resolved since we've added ytm as a provider.

@JellyBrick
